### PR TITLE
Backbone Module Structure

### PR DESCRIPTION
**Pull request by jacobcs**
An example backbone module structure for work in https://nexus.connectsolutions.com/display/PROJ/Backbone+module+structure

### CODE REVIEW

File: js/examples/backbone/app/mymodule/js/templates/example.ejs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div>
```
Comment (andrew-cs) [inline js/examples/backbone/app/mymodule/js/templates/example.ejs]: I'd like to see <div class="mymodule"> here for style containment

Comment (jacobcs) [inline js/examples/backbone/app/mymodule/js/templates/example.ejs]: +1

File: js/examples/backbone/app/mymodule/js/router.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        initialize: function() {
+            this.exampleCollection = new ExampleCollection();
+        },
+
+        /**
+        * Sets up the navigation area display for usage.
+        */
+        showNav: function() {
+            var nav = new NavView();
+            csx.navContainer.setView(nav);
```
Comment (andrew-cs) [inline js/examples/backbone/app/mymodule/js/router.js]: csx the global js variable is being removed, correct? Is this here as a placeholder / reminder?

Comment (jacobcs) [inline js/examples/backbone/app/mymodule/js/router.js]: @andrew-cs as mentioned in the meeting. I think the nav piece needs to be thought out and formalized. This is how we do it now

**Comment by andrew-cs** [comment]
+1

File: js/examples/backbone/app/mymodule/js/router.js (hunk 2 above)
Comment (nickcs) [inline js/examples/backbone/app/mymodule/js/router.js]: @jacobcs @andrew-cs if you review my POC you will see an example of my recommendation on how the nav is managed using a nav registration process.  However, my example doesn't user a view per nav element.  So we will need to combine these ideas once we actually have it developed in csx.  I am good with this for now.

File: js/examples/backbone/app/mymodule/js/router.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        },
+
+        /**
+        * Displays the main view.
+        */
+        showExampleView: function() {
+            this.showNav();
+            var view = new ExampleView({ collection: this.exampleCollection });
+            csx.container.setView(view, {emptyDOM: true});
```
Comment (nickcs) [inline js/examples/backbone/app/mymodule/js/router.js]: this is also different in my POC where I don't use a global csx.  But okay for now

**Pull request merged**